include/deal.II/lac/block_linear_operator.h

```diff
@@ -77,7 +242,7 @@ block_operator(const std::array<std::array<LinearOperator<typename Range::BlockT
     v.reinit(m);
 
     // And reinitialize every individual block with reinit_range_vectors:
-    for (unsigned int i = 0; i < m; ++i)
+    for (size_t i = 0; i < m; ++i)
```

Comment:
(I didn't say in my original question, but the point of asking was because you are -- probably unnecessarily -- doing 64-bit arithmetic when you use `std::size_t`, whereas `unsigned int` is 32-bit. In practice, I'm sure it makes very little difference other than possibly taking up a small number of extra bytes.)